lib/util/middleware.js

```diff
@@ -30,9 +31,9 @@ export async function secureFetch (url, accessToken, apiKey, method = 'get', opt
   }
 }
 
-export async function fetchUser (middlewareConfig, token, id) {
+export async function fetchUser (middlewareConfig, token) {
   const { apiBaseUrl, apiKey } = middlewareConfig
-  const requestUrl = `${apiBaseUrl}${API_USER_PATH}/find/auth0UserId/${id}`
+  const requestUrl = `${apiBaseUrl}${API_AUTH0USER_PATH}`
```

Comment:
If this is the only use of this constant, it probably isn't worth putting in a variable that's at the top of the file.

Reply:
@binh-dam-ibigroup, the point of having the `API_USER_PATH` constant is so that we can reuse it in functions in this file. So, there's really no need to have all of these interim variables. I think the clearest/most maintainable way to write this is just:

```javascript
return secureFetch(`${apiBaseUrl}${API_USER_PATH}/fromtoken`, token, apiKey)
```

or if that string is too long, just put it in `requestUrl`

Reply:
Good idea @landonreed, it is now:
```
  const requestUrl = `${apiBaseUrl}${API_USER_PATH}/fromtoken`
  return secureFetch(requestUrl, ...
```